timescaledb/how-to-guides/continuous-aggregates/time.md

```diff
@@ -85,26 +83,33 @@ minutes.
 
     ```sql
     SELECT create_hypertable('devices', 'time',
-      chunk_time_interval => 1000);
+      chunk_time_interval => 10);
     ```
 
 </procedure>
 
-To define a continuous aggregate on a hypertable that uses integer-based time, you need to have a function to get the current time in the correct format, and set it for the hypertable. This is done using the [`set_integer_now_func`][api-set-integer-now-func]. It can be defined as a regular PostgreSQL function, but needs to be [`STABLE`][pg-func-stable], take no arguments, and return an integer value of the same type as the time column in the table. When you have set up the time-handling, you can create the continuous aggregate.
+To define a continuous aggregate on a hypertable that uses integer-based time,
+you need to have a function to get the current time in the correct format, and
+set it for the hypertable. You can do this with the
+[`set_integer_now_func`][api-set-integer-now-func]
+function. It can be defined as a regular PostgreSQL function, but needs to be
+[`STABLE`][pg-func-stable],
+take no arguments, and return an integer value of the same type as the time
+column in the table. When you have set up the time-handling, you can create the
+continuous aggregate.
 
 <procedure>
 
 ### Creating a continuous aggregate with integer-based time
 
-1.  At the `psql` prompt, set up a function to convert the time to the FFF system:
+1.  At the `psql` prompt, set up a function to convert the time to the Unix epoch:
 
     ```sql
-    CREATE FUNCTION current_microfortnight() RETURNS BIGINT
+    CREATE FUNCTION current_epoch() RETURNS BIGINT
     LANGUAGE SQL STABLE AS $$
-    SELECT CAST(1209600 * EXTRACT(EPOCH FROM CURRENT_TIME) / 1000000 AS BIGINT)
-     $$;
+    SELECT EXTRACT(EPOCH FROM CURRENT_EPOCH)::bigint;
```

Comment:
Need the closing quotes after the function:
SELECT EXTRACT(EPOCH FROM CURRENT_EPOCH)::bigint;
$$